deploy/Dockerfile

```diff
@@ -5,6 +5,7 @@ WORKDIR /registry
 COPY deploy/olm-catalog .
 
 USER root
+RUN rm -f community-kubevirt-hyperconverged/icon.svg.b64
```

Comment:
Can we add this file to dockerignore, instead of removing it? Removing a file in a docker image, doesn't really remove it from previous layers.

Reply:
done, I hope it will not have side effects on other flows